templates/representatives/representative_detail.html

```diff
@@ -0,0 +1,49 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<div class="row">
+  <div class="col-sm-3">
+	<img src="{{ representative.photo }}">
+  </div>
+
+  {% include "representatives/_representative_block.html" %}
+  
+  <!--
+	  <div class="col-sm-3 text-center">
+	    <p>
+		  There is an ungoing &pi;Phone campaign, call this MEP for free to help La Quadrature!</p>
+	    <div class="btn-group" role="group">
+		  <button class="btn btn-large btn-primary" data-toggle="tooltip" data-placement="bottom" title="Call this MEP with the &pi;Phone"><span class="glyphicon glyphicon-phone-alt"></span></button>
+		  <a class="btn btn-large btn-default"><span class="glyphicon glyphicon-plus"></span> info</a>
+	    </div>
+	  </div>
+      -->
```

Comment:
Could you add a comment explaining why this is commented please ? ie. "Design for future feature is ready, leaving it here until we develop the Piphone campaign system".